cms/envs/common.py

```diff
@@ -713,6 +713,7 @@
 CSRF_TRUSTED_ORIGINS = []
 
 #################### CAPA External Code Evaluation #############################
+XQUEUE_WAITTIME_BETWEEN_REQUESTS = 5  # seconds
```

Comment:
It would be cleaner to store it in settings.XQUEUE_INTERFACE, but yes, we're still using this setting: https://github.com/edx/edx-platform/blob/557328a70eda0d58d10be132e226875a31c2825e/common/lib/xmodule/xmodule/capa_module.py#L1749-L1753

Reply:
Got it. 👍🏽